src/pip/_internal/operations/prepare.py

```diff
@@ -502,9 +497,8 @@ def prepare_linked_requirement(
                     )
                     if not os.path.exists(download_location):
                         shutil.copy(local_file.path, download_location)
-                        logger.info(
-                            'Saved %s', display_path(download_location)
-                        )
+                        download_path = display_path(download_location)
+                        logger.info('Saved %s', download_path)
```

Comment:
At https://github.com/pypa/pip/pull/8411/files#diff-e8eca5e33f546b893bee9f5abc68611bL393 , we are directly calling `display_path` in `logger.info`, but here we are assigning it to a variable and then passing the variable?

If this was just to make flake8 happy in terms of line-length, I don't think it's worth it, and we can stick with what was done previously

Reply:
I think the reasoning for the line-length limit is that if a statement is too long, it's probably doing too many things.  For this case in particular I don't think the difference is major though.

Reply:
In that case, I would personally prefer the original version, although I don't feel strongly about it.

```python
 logger.info(
    'Saved %s', display_path(download_location)
)
```

Reply:
Either works, and since no one feels strongly either way; let's keep what's in the PR right now. :)